beetmoverscript/utils.py

```diff
@@ -179,14 +179,17 @@ def _generate_beetmover_template_args_maven(task, release_props):
     }
 
     # Change version number to major.minor.buildId because that's what the build task produces
-    payload_version = FirefoxVersion.parse(task['payload']['version'])
+    payload_version = MavenVersion.parse(task['payload']['version'])
```

Comment:
@JohanLorenzo I'm interested particularly of your opinion here.

Reply:
It makes sense to me. That's a great change. `FirefoxVersion` was there before `MavenVersion` existed. The function is about maven stuff so yeah, let's use it.